qemu/tests/timedrift_check_when_hotplug_vcpu.py

```diff
@@ -0,0 +1,67 @@
+import re
+import time
+import logging
+
+from avocado.utils import process
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Check time offset after hotplug a vcpu.
+
+    1) sync host time with ntpserver
+    2) boot guest with '-rtc base=utc,clock=host,driftfix=slew'
+    3) stop auto sync service in guest (rhel7 only)
+    4) sync guest system time with ntpserver
+    5) hotplug a vcpu by qmp command
+    6) query guest time offset with ntpserver for several times
+
+    :param test: QEMU test object.
+    :param params: Dictionary with test parameters.
+    :param env: Dictionary with the test environment.
+    """
+
+    error_context.context("Sync host system time with ntpserver", logging.info)
+    process.system("ntpdate %s" % params["ntp_server"])
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    timeout = int(params.get("login_timeout", 360))
+    session = vm.wait_for_login(timeout=timeout)
+
+    ntp_query_cmd = params.get("ntp_query_cmd", "ntpdate -q clock.redhat.com")
+    query_times = int(params.get("query_times", "4"))
+    query_internal = float(params.get("query_internal", "600"))
+    drift_threshold = float(params.get("drift_threshold", "3"))
+
+    error_context.context("Stop auto sync service in guest", logging.info)
+    cmd = "(service chronyd status | grep 'Loaded: loaded')"
+    cmd += " && service chronyd stop && service ntpd stop"
+    session.cmd_status_output(cmd)
+
+    error_context.context("Sync time from guest to ntpserver", logging.info)
+    session.cmd("yum install -y ntpdate; ntpdate clock.redhat.com",
```

Comment:
params["ntp_server"]  instead.

Reply:
@vivianQizhu Wait for your answer.

Reply:
@sitoliu By this I mean use a param for the command line, instead of hard code it.